src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -138,10 +138,16 @@
         </property>
        </widget>
       </item>
+      <item row="1" column="0">
+       <widget class="QCheckBox" name="checkBox_SeratoMetadataExport">
+        <property name="text">
+         <string>Write Serato Metadata to files (experimental/potentially makes files unusable in Serato!)</string>
```

Comment:
```suggestion
         <string>Write Serato Metadata to files</string>
```
I suggest replacing the ambiguous disclaimer with a confirmation dialog when the user enables the preference that says something like "Writing Serato metadata to files is experimental. It is recommended that you back up your music files before enabling it. Are you sure you want to enable this experimental feature?"